packages/client/src/anonymous.tsx

```diff
@@ -179,6 +180,62 @@ function UserRegistration() {
   );
 }
 
+// .code-box-demo {
+//   padding: 42px 24px 50px;
+//   color: rgba(0,0,0,.65);
+//   border-bottom: 1px solid #ebedf0;
+// }
+
+function AuthorizationCode() {
+  const code = window.apiTokenExhangeCode;
+  const [copyStatus, copy] = useClipboard({ lazy: true, timeout: 2000 });
+  return (
+    <div>
+      <div className='header_container' style={{ backgroundColor: '#373d62' }}>
+        <Logo />
+      </div>
+      <div
+        style={{
+          display: 'flex',
+          justifyContent: 'center',
+          alignItems: 'center',
+          minHeight: 400,
```

Comment:
I think adding `px` might be better if it will not break your UI.

```suggestion
          minHeight: '400px',
```